apps/admin/traefik2-auth-middlewares/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Has this been added to the environment kustomization anywhere? similar to https://github.com/hmcts/cnp-flux-config/blob/master/apps/admin/demo/base/kustomization.yaml#L18